icub/conf/gazebo_icub_right_hand_pinky.ini

```diff
@@ -19,29 +19,38 @@ pinky_control (0 1 2 3 4 5) (r_pinky reserved reserved reserved reserved reserve
 # name of the device to be instatiated by the factory
 device gazebo_controlboard
 #jointNames list
-jointNames    r_lij3 r_lij4 r_lij5 r_rij3 r_rij4 r_rij5    
+jointNames r_lij3 r_lij4 r_lij5 r_rij3 r_rij4 r_rij5
 name right_hand_pinky
 
 #PIDs:
 # this information is used to set the PID values in simulation for GAZEBO, we need only the first three values
-[GAZEBO_PIDS]
-Pid0  100.0 0.01 10.0 9999 9999 9 9
-Pid1  100.0 0.01 10.0 9999 9999 9 9
-Pid2  100.0 0.01 10.0 9999 9999 9 9
-Pid3  100.0 0.01 10.0 9999 9999 9 9
-Pid4  100.0 0.01 10.0 9999 9999 9 9
-Pid5  100.0 0.01 10.0 9999 9999 9 9
-
-[GAZEBO_VELOCITY_PIDS]
-Pid0  500.0 2.0 0.1  9999 9999 9 9
-Pid1  500.0 2.0 0.1  9999 9999 9 9
-Pid2  500.0 2.0 0.1  9999 9999 9 9
-Pid3  500.0 2.0 0.1  9999 9999 9 9
-Pid4  500.0 2.0 0.1  9999 9999 9 9
-Pid5  500.0 2.0 0.1  9999 9999 9 9
+[POSITION_CONTROL]
+controlUnits  metric_units
+controlLaw    joint_pid_gazebo_v1
+kp            1.745 1.745 1.745 1.745 1.745 1.745
+kd            0.0   0.0   0.0   0.0   0.0   0.0
```

Comment:
@Yeshasvitvs Have you check that setting this `kd` to zero (they where different from zero before) is not affecting the behaviour of the simulation?